src/cmd/namespaces/update/update-namespace.go

```diff
@@ -30,12 +25,9 @@ var UpdateNamespaceCmd = &cobra.Command{
 		}
 
 		id := args[0]
-		r, err := c.UpdateNamespaceMutationWithResponse(ctxTimeout,
-			id,
-			cloudapi.UpdateNamespaceMutationJSONRequestBody{
-				EnvironmentId: lo.Ternary(viper.IsSet(EnvironmentIDKey), lo.ToPtr(viper.GetString(EnvironmentIDKey)), nil),
-			},
-		)
+		environmentID := args[1]
```

Comment:
I'd keep the `--env-id` flag, in-line with our other update subcommand (e.g. `otterize env update add-label <env-id> --key asd ...`). This keeps our CLI structure clearer - all the single-object-update commands accept a single argument (object id), and flags, some required and some optional. Don't forget to mark the environment id flag as required.